src/models/OtrStore.js

```diff
@@ -0,0 +1,67 @@
+var _ = require("lodash");
+var DSA = require('otr').DSA;
+var OTR = require('otr').OTR;
+var Chan = require("./chan");
+
+module.exports = OtrStore;
+
+/** Stores OTR state for a given user
+ *
+ * Handels private key, current sessions and fingerprints.
+ */
+function OtrStore(name, client) {
+	_.merge(this, {
+		name: name,
+		privKey: undefined,
```

Comment:
Mmh, forget what I said about JS oddities, it doesn't apply here anyway, my bad. Forget about being explicit or implicit as well, this has nothing to do. What you are doing here is essentially:

```js
this.privKey = undefined;
// ...
this.privKey = new DSA();
```

This is just confusing and adds no readability: say the function goes more complex, one will wonder why the private key defaults to `undefined`.
Conclusion: either remove the line from the hash, or replace `undefined` with `new DSA();` within the hash. You can still move your `console.log()` test above that in the second solution. But one needs to go.